README.md

```diff
@@ -69,3 +69,17 @@ TODO:
 - More testing
 - More filter types
 - more?
+
+## Citation
+
+The preferred BibTeX entry for citation of `ads2inspire` is
+
+```
+@software{ads2inspire,
+  author = "{Stein, Leo C. and Feickert, Matthew}",
```

Comment:
@duetosymmetry I'm not very good at remembering how middle initials work in BibTeX citations and I didn't check this, but as I think you have plenty of experience here I'll let you tell me if this needs correcting.

Reply:
Verified